migration/ram.c

```diff
@@ -2123,28 +2129,30 @@ static int ram_state_init(RAMState **rsp)
     qemu_mutex_init(&(*rsp)->src_page_req_mutex);
     QSIMPLEQ_INIT(&(*rsp)->src_page_requests);
 
-    /*
-     * Count the total number of pages used by ram blocks not including any
-     * gaps due to alignment or unplugs.
-     */
-    (*rsp)->migration_dirty_pages = ram_bytes_total() >> TARGET_PAGE_BITS;
-
     ram_state_reset(*rsp);
 
     return 0;
 }
 
-static void ram_list_init_bitmaps(void)
+static void ram_list_init_bitmaps(RAMState *rs)
```

Comment:
check if `rs` is NULL

Reply:
If that is possible, the original code would panic at https://github.com/kata-containers/qemu/pull/2/files#diff-7a5fd992f4559f6907ed9fd0a89cd159R839